packages/ckeditor5-coremedia-differencing/src/Differencing.ts

```diff
@@ -63,40 +63,37 @@ export default class Differencing extends Plugin {
     const { model, conversion } = editor;
     const { schema } = model;
 
-    schema.register("xdiff-span", {
-      allowIn: ["$block", "$container"],
-      allowChildren: ["$text", "$block", "$container"],
+    schema.register("xdiffSpan", {
```

Comment:
Note for review: Changes to CamelCase namespace similar to CKEditor's GHS, where we have htmlPre, htmlTable, etc.